packages/documentation/src/pages/frontend-support-dashboard/containers/CommitsTable.jsx

```diff
@@ -0,0 +1,139 @@
+  const trueIcon = <span style={{color: "green"}} aria-label="true"><i class="fas fa-check-circle fa-3x" aria-hidden="true"></i></span>;
+  const falseIcon = <span style={{color: "red"}} aria-label="true"><i class="fas fa-times-circle fa-3x" aria-hidden="true"></i></span>;
+
+  return isOn ? trueIcon : falseIcon;
+}
+
+export default function CommitsTable({
+  repo,
+  devBuildText,
+  stagingBuildText,
+  prodBuildText,
+  commits,
+}) {
+  const devRows = devBuildText.split('\n').filter(x => x) || [];
+  const stagingRows = stagingBuildText.split('\n').filter(x => x);
+  const prodRows = prodBuildText.split('\n').filter(x => x);
+
+  const devRef = devRows[6]?.slice(4);
+  const stagingRef = stagingRows[6]?.slice(4);
+  const prodRef = prodRows[6]?.slice(4);
+  let isOnDev = false;
+  let isOnStaging = false;
+  let isOnProd = false;
+
+  const openAccordion = repo.repo === 'vets-website' ? false : true;
+
+  return (
+    <div>
+
+      <div class="feature">
+        <h3>{repo.repo} BUILD.txt files</h3>
+        <div style={{ display: 'flex', justifyContent: 'space-around', fontSize: '0.85em' }}>
+            <div>
+              <h4>
+                <a href={repo.devBuildText}>Dev</a>
+              </h4>
+              {devRows.map(info => {
+                return <div key={info}>{info}</div>;
+              })}
+            </div>
+            <div>
+              <h4>
+                <a href={repo.stagingBuildText}>Staging</a>
+              </h4>
+              {stagingRows.map(info => {
+                return <div key={info}>{info}</div>;
+              })}
+            </div>
+            <div>
+              <h4>
+                <a href={repo.prodBuildText}>Prod</a>
+              </h4>
+              {prodRows.map(info => {
+                return <div key={info}>{info}</div>;
+              })}
+            </div>
+          </div>
+      </div>
+
+
+      <va-accordion multi bordered>
+        <va-accordion-item 
+          header={`${repo.repo} commits`} 
+          subheader='latest 30 commits'
+          open={openAccordion}
+        >
+          <table>
+            <thead>
+              <tr>
+                <th>Author</th>
+                <th>Commit</th>
+                <th>On Dev?</th>
+                <th>On Staging?</th>
+                <th>On Prod?</th>
+              </tr>
+            </thead>
+            <tbody>
+              {commits.map(x => {
+                const { commit = {}, author: githubAuthor = {}, html_url, sha, } = x; // eslint-disable-line camelcase
+                const { author = {}, committer = {}, message = '' } = commit;
+                const { name } = author;
+                const { date } = committer;
+                const { login } = githubAuthor;
+
+                if (sha === devRef) isOnDev = true;
+                if (sha === stagingRef) isOnStaging = true;
+                if (sha === prodRef) isOnProd = true;
+
+                const imageStyle = {
```

Comment:
I think this is where the CSS would live: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/master/packages/documentation/src/scss